xml/vt_scenarios.xml

```diff
@@ -0,0 +1,198 @@
+   <itemizedlist>
+    <listitem>
+     <para>
+      Maintenance windows should be carefully planned
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Storage is a key point to be able to support migration and growing disk usage
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Server capabilities must be checked to support the new workload
+     </para>
+    </listitem>
+   </itemizedlist>
+  </important>
+ </sect1>
+ <sect1>
+  <title>Isolation</title>
+  <para>
+   A guest operating system can be fully isolated from the host
+   running it. So if the virtual machine is corrupted, the host system is
+   not harmed. It also does not affect any other virtual machines. No data
+   is shared between VM.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Secure Boot can be used for VM
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     KSM should be avoided
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     CPU can be assigned to a VM, and HT should be disabled to avoid security issue
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     VM should not share network / storage / network / Hardware
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Migration capabilities could be impacted in case of specific usage like PCI passthrough or NUMA
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Full Virtualization is generally less efficient if no PV/virtio drivers are used
+    </para>
+   </listitem>
+  </itemizedlist>
+<para>
+    AMD provides some specific features regarding Virtualization security.
+    <!-- link to the SEV doc? -->
+   </para>
+  <itemizedlist>
+   
+   <listitem>
+    <para>
+     AMD Secure Memory Encryption (SME):
+     Uses a single key to encrypt system memory. The key is generated by the AMD Secure Processor at boot. SME requires enablement in the system BIOS or operating system. When enabled in the BIOS, memory encryption is transparent and can be run with any operating system.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     AMD Secure Encrypted Virtualization (SEV):
+     Uses one key per virtual machine to isolate guests and the hypervisor from one another. The keys are managed by the AMD Secure Processor. SEV requires enablement in the guest operating system and hypervisor. The guest changes allow the VM to indicate which pages in memory should be encrypted. The hypervisor changes use hardware virtualization instructions and communication with the AMD Secure processor to manage the appropriate keys in the memory controller.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     AMD Secure Encrypted Virtualization-Encrypted State (SEV-ES):
+     Encrypts all CPU register contents when a VM stops running. This prevents the leakage of information in CPU registers to components like the hypervisor, and can even detect malicious modifications to a CPU register state.
```

Comment:
```suggestion
     Since SEV only operates when a VM is running, the additional SEV-ES feature encrypts
     all CPU register contents when a VM stops running. This prevents the leakage of information
     in VMs' CPU registers to the hypervisor. SEV-ES can also detect malicious modifications to
     a CPU's register state.
```